### PR TITLE
feat: add Team Carousel Component

### PR DESCRIPTION
**Pull request by swaindhruti**
This PR adds a responsive Team Carousel component with marquee effect.

### CODE REVIEW

File: src/components/teamMembers/teamCards.styles.jsx
Comment (anmol5936) [inline src/components/teamMembers/teamCards.styles.jsx:1]: add aspect ratio to images. Some images are not clear and are getting outside of the circle

Comment (anmol5936) [inline src/components/teamMembers/teamCards.styles.jsx:1]: Add left margin as specified in the design

File: src/components/teamMembers/teamCarousel.jsx
Comment (anmol5936) [inline src/components/teamMembers/teamCarousel.jsx:1]: Meet the team should be in a single line from smaller screen

**Review by anmol5936 — CHANGES_REQUESTED** [review]

File: src/components/teamMembers/teamCards.styles.jsx
Comment (swaindhruti) [reply src/components/teamMembers/teamCards.styles.jsx:1]: added margin to both the sides of the carousel

File: src/components/teamMembers/teamCarousel.jsx
Comment (swaindhruti) [reply src/components/teamMembers/teamCarousel.jsx:1]: made the heading responsive

File: src/components/teamMembers/teamCards.styles.jsx
Comment (swaindhruti) [reply src/components/teamMembers/teamCards.styles.jsx:1]: added aspect ratio to the image

File: src/config/content/mycomp_data/team.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+export const TeamMembers = [
```
Comment (Srish-ty) [inline src/config/content/mycomp_data/team.js]: instead of writing in `mycomp_data`, create a dir `content/teamsData/`

**Review by Srish-ty — CHANGES_REQUESTED** [review]
remove `pnpm-lock.yaml`
or git rm pnpm-lock.yaml

File: src/config/content/mycomp_data/team.js (hunk 1 above)
Comment (swaindhruti) [reply src/config/content/mycomp_data/team.js]: Added to the specified path

**Comment by anmol5936** [comment]
@Shurtu-gal  LGTM 🚀
Please proceed with merging

**Review by anmol5936 — APPROVED** [review]

**Pull request merged**